packages/react-components/src/components/styles/ControlBar.styles.ts

```diff
@@ -128,8 +128,7 @@ export const controlButtonStyles: IButtonStyles = {
     border: 'none'
   },
   flexContainer: {
-    flexFlow: 'column',
-    display: 'contents'
+    display: 'unset'
```

Comment:
Is this change intentional for the lock file updates?

Reply:
yes, it is intentional. The update lock files requires this change.

Reply:
All visual regression tests are passing. So we are good. Also from testing locally, things are looking good.